src/sb-util.ts

```diff
@@ -282,6 +302,14 @@ export class Block extends BlockCollection {
         if (!props) return null;
         return props[property];
     }
+
+    public parent(): Block {
+        const parentId: string = this.prop('parent');
+        if (parentId) {
+            return new Sprite(getBlockMeta(this.props()).sprite).blocks().byId(parentId);
```

Comment:
Though for the next revision: Maybe we save the project and lookup sprite on demand the first time? That would let us use the same `Sprite` instance. Hmm. How could do that without making a new Project as well.

We may need to look up plenty of things, and they'll all be under the project. But not necessarily under the Sprite. Like if we wanted a sound asset. In that case if we saved the project and didn't know if we would need the sprite we could skip saving the sprite initially and look up the sprite on demand.